docs/write-side.md

```diff
@@ -21,10 +21,27 @@ Aggregate state is explicitly passed to all of these functions as an argument.
 
 ## Aggregate ID
 
-Each aggregate should have a unique ID that is immutable during the aggregate's lifetime. An Aggregate ID should stay unique in the given event store, however we also recommend to keep it
-globally unique. We recommend that you use [UUID v4](https://github.com/kelektiv/node-uuid#version-4) or [cuid](https://github.com/ericelliott/cuid) to generate aggregate IDs for scalable apps.
+Each aggregate instance should have a unique immutable ID. You should generate an aggregate ID on the client and send it to reSolve with a command that creates a new aggregate:
```

Comment:
send it to reSolve with a command that creates a new aggregate:
??? send it to resolve?

Reply:
"reSolve" is the name of the framework. You can read it as: "send it to the reSolve framework".

Reply:
ok